### DISCUSSION TITLE
Silent Mode with costum values

### DISCUSSION THREAD

**Discussion by SmikeSix** [BartoszCichecki/LenovoLegionToolkit#946] [Q&A] [answered]
Hi i have a Lenovo Legion 5 with a 7745hx & 4060.
I was playing around with performance settings and i found that silent mode is overall the best mode, especially when it comes to silence / energy usage and performance. My CPU on silent mode is only 5% less performant than max performance, i score really high. The laptop is very loud otherwise.

The only issue is that the GPU falls low on that mode with a 15-25% performance penalty. 

With costum settings and a minimized fan curve the laptop still gets loud. No matter how low i set all those settings. It probably is even the CPU that triggers that. Yeah just tested it, its under CPU load that fans go up.

Is there a way to tweak the silence settings?  It really is the sweet spot for me CPU wise, but GPU wise i think it can go a heigher. 
Thanks a lot and great tool!

**Comment by BartoszCichecki** [comment, marked as answer]
Answer to this question is already there in the FAQ.
